src/components/pages/dashboard-pages/EditProfileForm.jsx

```diff
@@ -0,0 +1,71 @@
+import FormInput from '../../common/FormInput';
+import SelectDropdown from '../../common/SelectDropdown';
+
+const EditProfileForm = ({ options, role, cancel }) => {
+  // this is just a layout without sending the data to DB and storing the state
+  return (
+    <div className="w-full flex flex-grow items-center sm:items-start sm:justify-center sm:flex-row flex-col overflow-y-hidden">
+      <div className="sm:w-1/5 w-4/5 flex flex-col items-center gap-4 mt-4 p-4">
+        {/* div for user's photo */}
+        <div className="h-24 w-24 bg-grey rounded-full"></div>
+        <button className="bg-red hover:bg-pureWhite hover:text-red h-8 w-1/3 sm:w-2/4 hover:border-2 hover:border-red text-white font-spartan font-semibold text-base sm:text-lg rounded-lg transition duration-300 easy-in">
+          Add Photo
+        </button>
+      </div>
+      <form className="mt-4 p-4 sm:w-2/5 w-4/5 h-full">
+        <div className="flex gap-8">
+          <FormInput placeholder=" " name="firstName">
+            First name
+          </FormInput>
+          <FormInput placeholder=" " name="lastName">
+            Last name
+          </FormInput>
+        </div>
+        <FormInput placeholder=" " name="email">
+          Email
+        </FormInput>
+        {role === 'teacher' ? (
+          <FormInput placeholder=" " name="educationExperience">
+            Education & Experience
+          </FormInput>
+        ) : (
+          <></>
```

Comment:
Would conditional rendering work better here?

Reply:
Did you mean that we need to have two forms for editing profile: the first one for teacher and the second one for student?

Reply:
No just some conditional rendering instead of the fragment .